packages/backend/src/graphql/resolvers/peer.test.ts

```diff
@@ -13,14 +13,13 @@ import { initIocContainer } from '../..'
 import { Config } from '../../config/app'
 import { truncateTables } from '../../tests/tableManager'
 import { errorToCode, errorToMessage, PeerError } from '../../peer/errors'
-import { Peer as PeerModel } from '../../peer/model'
+import { Peer, Peer as PeerModel } from '../../peer/model'
```

Comment:
Why do you need the model twice?

Reply:
You can't have a peer without being a peer.
There's always at least two.